src/edu/uwm/cs/util/Reach.scala

```diff
@@ -1,4 +1,4 @@
-/* Dtermining which nodes are in cycles in a graph.
+/* Determining which nodes are in cycles in a graph.
```

Comment:
The comment before and after seems to be wrong.  It was copied from Cycle.scala, but this file doesn't do cycle detection, does it?